### PR TITLE
proper distances in the plane of view

### PR DESCRIPTION
**Pull request by bazkiaei**
This PR is trying to solve the first part of issue #54.
Pynbody provides the position of particles in units of `Mpc a / h`. It is easy to convert this data to `Mpc`, but the result will be comoving distances. The code is able to convert comoving distances to luminosity distances which are desired in the viewing direction, but in the viewing plane, it should be physical distances, which is the same as proper distances (please correct me if I am wrong!). The difference between proper distance and comoving distance is `a`, the cosmological scale factor. So, instead of converting to `Mpc`, if the code converts the distances in the viewing plane to `Mpc a`, then the result is the proper distance which is the desired one.

### CODE REVIEW

File: mocks/mocks.py [hunk 1]
```diff
@@ -155,20 +155,24 @@ def read_gadget(config):
...
     sim_properties : pynbody.simdict.SimDict
         Contains cosmological information of the simulation.
     """
     sim_data = pynbody.load(config['sim_data_path'])
-    particle_pos = sim_data.stars['pos'].in_units('Mpc')
-    particle_pos = particle_pos.astype(np.float64)
+    particle_pos = sim_data.stars['pos']
+    # Converting the distances in viewing axis to co-moving distances and
+    # distances in viewing plane to proper distances.
+    desired_pos = convert_to_desired_distance(config,
```
Comment (bazkiaei) [inline mocks/mocks.py]: There is another option to do this outside of this function. I appreciate any advice about that which way is better.

**Comment by bazkiaei** [comment]
@AstroHuntsman/spiderlings 
Ready for review.

File: mocks/mocks.py (hunk 1 above)
Comment (lspitler) [reply mocks/mocks.py]: Probably finet o leave for now.

File: mocks/mocks.py [hunk 2]
```diff
@@ -938,6 +942,48 @@ def convert_mass_to_light(config,
     return mass
 
 
+def convert_to_desired_distance(configuration,
```
Comment (lspitler) [inline mocks/mocks.py]:
```suggestion
def convert_pynbody_to_desired_distance(configuration,
```

**Review by lspitler — APPROVED** [review]
Looks good @bazkiaei

**Pull request merged**